src/extensions/default/BrambleUrlCodeHints/camera/selfieWidget.html

```diff
@@ -8,8 +8,29 @@
     </div>
     <div id="selfie-gallery">
         <div class="polaroid">
+        <!--<div>-->
             <img id="selfie-photo" />
```

Comment:
Probably what we are going to want to do, is replace this image (i.e., visually in the page) with the canvas you'll render into with Caman.  We may or may not need this `img` at all, depending on how this code functions (I have to re-read it to remember).  We want to share the canvas Caman is using with that's used internally.